src/main/java/uk/gov/hmcts/reform/sscs/service/CoverLetterService.java

```diff
@@ -69,11 +69,11 @@ private void printCoverLetterToPdfLocallyForDebuggingPurpose(byte[] coverLetterC
     }
 
     public byte[] generateCoverLetter(SscsCaseData caseData, FurtherEvidenceLetterType letterType, String templateName,
-                                      String hmctsDocName) {
+                                      String hmctsDocName, String otherPartyId) {
```

Comment:
Would be nice here as well to reduce the number of parameters

Reply:
Can't see easy way of doing this